### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by luisv01**
!

### CODE REVIEW

**Comment by luisv01** [comment]
!

File: index.html [hunk 1]
```diff
@@ -115,6 +115,31 @@ <h2>A project for first-time contributions</h2>
           <!-- Keep one line of space above and below your card -->
           <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+         <!-- ---------- luisv001 card SART-------- -->
+          <div class="card">
+            <p class="name">Luis Villa</p>
+            <p class="contact">
+                <i class="fab fa-github"></i>
+                <a href="https://github.com/luisv01" target="_blank">Your handle</a>
```
Comment (TenTraicion) [inline index.html:123]:
```suggestion
                <a href="https://github.com/luisv01" target="_blank">@luisv01</a>
```

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@luisv01, you need to add your social handle in the code.

**Comment by luisv01** [comment]
> @luisv01, you need to add your social handle in the code.

@TenTraicion thanks for catching that. Applied your suggestion!

**Review by luisv01 — COMMENTED** [review]
@TenTraicion all set, thanks for the comment!

**Review by TenTraicion — APPROVED** [review]

**Comment by TenTraicion** [comment]
@luisv01, Congratulation on your PR Merge, and Thanks for your Contribution. You can check your card in our [Live Website](https://syknapse.github.io/Contribute-To-This-Project/)! It may take 2-3 minutes to make your card visible in our website. 

Welcome to Open Source Community. You can follow the [Next Steps](https://github.com/Syknapse/Contribute-To-This-Project#next-steps) now. You are also welcome to join our [Discord Server: https://discord.gg/tWkvS4ueVF](https://discord.gg/tWkvS4ueVF) and be a part of our community.

**Pull request merged**